CHANGELOG.md

```diff
@@ -5,6 +5,41 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+ * [`changed`] `base_config.inc` to work with I2C and SHDLC at the same time.
+               Changed HAL implemantation locations and moved the UART one from
+               the Makefile of the test in the sensor repo to here. Also changed
+               the way how mux sources are included so they are only needed by i2c
+               tests.
+ * [`added`] SHDLC implementation. This implementation was immigrated from the
+             `embedded-uart-sps` repo. Since it's general code which could be used
+             by multiple sensors it makes more sense to interate it here. The
+             Shdlc part also uses the files in `common/`. In addition to that
+             it adds two `.[ch]` file pairs. First `sensirion_shdlc.[ch]` which
+             contains the SHDLC implementation for Sensirion sensors. Sencondly
+             `sensirion_uart_hal.h` and `sensirion_uart_implementation.c` which
+             describe the hardware abstraction layer used by `sensirion_shdlc.c`.
+             This hal can be implemented in `sensirion_uart_implentation.c` for
+             the platform the sensor will be used on.
+ * [`changed`] Structure of embedded-common. File names should be more logical
+               and the spliting of the different parts of the code as well.
+               Embedded-common now splites the code into three `.[ch]` file pairs
+               and a configuration `.h` file.
+               First there are the three files in `common/`. `sensirion_config.h`
+               is similar to `sensirion_arch_config.h` from before. It contains
```

Comment:
I think this is quite relevant to everyone upgrading, I'd make a separate `changed` entry just to say `rename sensirion_arch_config.h -> sensirion_config.h`

Reply:
Add that.